lua/gitblame/git.lua

```diff
@@ -118,6 +118,11 @@ end
 ---@param callback fun(url: string)
 function M.get_file_url(filepath, sha, line1, line2, callback)
     M.get_repo_root(function(root)
+        if root == "" then
+            callback(filepath)
+            return
+        end
```

Comment:
If outside of a repository, just return the filename so `xdg-open` and `open` might stil be able to open the file. Basically instead of outputting something like

```plaintext
/blob///Documents/my-file.txt#L1
```

it will output

```plaintext
/home/username/Documents/my-file.txt
```

Reply:
Would be great to add this as a comment

Reply:
Done in https://github.com/f-person/git-blame.nvim/pull/125/commits/7b8025282c2d3c9325de093ab54d3c913cf125dd